### PR TITLE
Move prev cash crop

### PR DESCRIPTION
**Pull request by Adamws33**
removed cash crop from the sidebar and placed it in the goals section of the stepper.

### CODE REVIEW

**Comment by mikahpinegar** [comment]
```
src\pages\CropSidebar\CropSidebar.js
  Line 38:46:  'updateDateRange' is defined but never used  no-unused-vars
```

File: src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js [hunk 1]
```diff
@@ -1,67 +1,68 @@
...
-                label="Harvest Date"
-                value={cashCropDataRedux.dateRange.endDate}
-                onChange={(newDate) => handleDispatch(cashCropDataRedux.dateRange.startDate, newDate)}
-              />
-            </LocalizationProvider>
-          </ListItem>
-        </List>
-      </Collapse>
+        <Typography variant="body2" align="center" color="secondary" gutterBottom>
+          CASH CROP GROWING WINDOW
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js:27]: can you change to `Enter your cash crop growing window if you'd like to see it displayed on the calendar.`

Comment (mikahpinegar) [reply src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js:27]: get rid of all caps

File: src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const PreviousCashCrop = () => {
       <Grid
         item
         xs={12}
-        style={{ paddingTop: '2rem' }}
+        style={{ paddingTop: '2rem', paddingBottom: '7px' }}
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js:24]: ted made a good suggestion to not add padding or margin and margin to Grids

File: src/pages/CropSidebar/CropSidebar.js [hunk 3]
```diff
@@ -73,10 +72,10 @@ const CropSidebar = ({
   const [sidebarFiltersData, setSidebarFiltersData] = useState([]);
   // const [tableHeight, setTableHeight] = useState(0);
   const [cropFiltersOpen, setCropFiltersOpen] = useState(true);
-  const [dateRange, setDateRange] = useState({
-    startDate: null,
-    endDate: null,
-  });
+  // const [dateRange, setDateRange] = useState({
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/CropSidebar.js:75]: remove commented out code

File: src/pages/CropSidebar/CropSidebar.js [hunk 4]
```diff
@@ -258,18 +257,17 @@ const CropSidebar = ({
...
-        dispatchRedux(updateDateRange({
-          startDate: dateRange.startDate.toISOString().substring(0, 10),
-          endDate: dateRange.endDate.toISOString().substring(0, 10),
-        }));
-      }
+  // useEffect(() => {
+  //   if (from === 'table') {
+  //     if (dateRange.startDate !== null && dateRange.endDate !== null) {
+  //       dispatchRedux(updateDateRange({
+  //         startDate: dateRange.startDate.toISOString().substring(0, 10),
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/CropSidebar.js:264]: remove

File: src/pages/CropSidebar/CropSidebar.js [hunk 5]
```diff
@@ -258,18 +257,17 @@ const CropSidebar = ({
...
+  //         endDate: dateRange.endDate.toISOString().substring(0, 10),
+  //       }));
+  //     }
 
-      setGrowthWindow(true);
-    }
-  }, [dateRange, from, setGrowthWindow]);
+  //   }
+  // }, [dateRange, from, setGrowthWindow]);
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/CropSidebar.js:270]: remove

File: src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js (hunk 2 above)
Comment (mikahpinegar) [reply src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js:24]: you can put padding on a box that contains a grid though

**Comment by mikahpinegar** [comment]
can you also move it to the location tab instead of the goals tab?

**Comment by mikahpinegar** [comment]
also there is a conflcit

File: src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js (hunk 2 above)
Comment (Adamws33) [reply src/pages/CropSidebar/PreviousCashCrop/PreviousCashCrop.js:24]: ahh didnt think about that i updated it.

**Comment by Adamws33** [comment]
I moved the component to the location window and made the other updates listed.

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**